### PR TITLE
fix(tools): bust element css browser cache

### LINKED ISSUE #2387: [pfe-tools]: dev-server doesn't reflect changes to .css files
## Description of the issue

When developing elements using `npm run start` changes to `.css` files aren't reflected in the browser.  You need to stop the dev-server and restart it for the new css to be seen.

### Impacted component(s)

- All components

### Steps to reproduce

1. `nvm use`
2. `npm run clean:nuke`
3. `npm run ci`
4. `npm run build`
5. `npm run start`
6. Visit: http://localhost:8000/components/label/demo/
7. In your IDE, edit the `elements/pf-label/pf-label.css` file and add a the following css
```css
:host {
  display: none;
}
```
8. Verify that the css change was not reflected on the demo page in your browser.

### Expected behavior

CSS changes should immediately seen in your browser.

## Related issues

- https://github.com/RedHat-UX/red-hat-design-system/pull/735

### PR DESCRIPTION
**Pull request by bennypowers**
Fixes #2387 

## What I did

1. update the version of web-dev-server-plugin-lit-css, after https://github.com/bennypowers/lit-css/pull/35
2. refactor the config file

## Testing Instructions

1. run the dev server, change some element css files

### CODE REVIEW

File: tools/pfe-tools/dev-server/config.ts [hunk 1]
```diff
@@ -52,6 +53,8 @@ function renderBasic(context: Context, demos: unknown[], options: PfeDevServerCo
   return env.render('index.html', { context, options, demos });
 }
 
+const md5 = createHmac('md5', 'trust no one');
```
Comment (brianferry) [inline tools/pfe-tools/dev-server/config.ts:56]: Is this variable used in anything?

Comment (brianferry) [reply tools/pfe-tools/dev-server/config.ts:56]: Same with `join` and `stat` on line 8 / 9?

**Review by brianferry — COMMENTED** [review]
Looks good to me locally, I edited the pf-spinner animation and it automatically updated

File: tools/pfe-tools/dev-server/config.ts (hunk 1 above)
Comment (bennypowers) [reply tools/pfe-tools/dev-server/config.ts]: thanks fixed. was using it to calculate etags but we don't actually need to

**Comment by kylebuch8** [comment]
I'm not sure how but I think this PR broke the patternflyelements.org site.

image

The JS error is preventing all the web component code from running.

**Pull request merged**